### DISCUSSION TITLE
RedisManager question!

### DISCUSSION THREAD

**Discussion by ovresko** [miguelgrinberg/python-socketio#756] [Q&A]
as it is mentioned in the docs regarding RedisManager, i can't find a way to listen to events in the external process 

External module B has RedisManager rm, i can rm.emit(something) but @rm.event and @rm.on doesn't work how can i listen to events inside external process 

> A limitation of the write-only client manager object is that it cannot receive callbacks when emitting. When the external process needs to receive callbacks, using a client to connect to the server with read and write support is a better option than a write-only client manager.

**Comment by miguelgrinberg** [comment]
The pubsub managers are not intended for listening, they are designed to publish events only. Listening to events is possible only in the server processes.
